### PR TITLE
Trading standards content fixes

### PR DESCRIPTION
**Pull request by edwardhorsford**
Update content primarily on the trading standards ts_investigations flow.

Changes:
* Update labels and hint text to make them clearer
* Add unique page titles to pages
* Cleanup grid markup where appropriate

### CODE REVIEW

File: psd-web/app/views/investigations/ts_investigations/_file_page.html.slim [hunk 1]
```diff
@@ -15,6 +15,8 @@ fieldset.govuk-fieldset
       = render "form_components/govuk_input", key: :title, form: subform, value: @file_title,
               classes: "govuk-!-width-two-thirds",
               label: { text: "Title" }
+      - file_description_hint = file_description_hint.presence ||\
```
Comment (slorek) [inline psd-web/app/views/investigations/ts_investigations/_file_page.html.slim]: `present?`

Comment (slorek) [reply psd-web/app/views/investigations/ts_investigations/_file_page.html.slim]: `file_description_hint ||= "For example, ‘Orange, 4 socket extension lead'’"

File: psd-web/app/views/investigations/ts_investigations/_file_page.html.slim [hunk 2]
```diff
@@ -15,6 +15,8 @@ fieldset.govuk-fieldset
...
+          "For example, ‘Orange, 4 socket extension lead'’"
       = render "form_components/govuk_textarea", key: :description, form: subform, value: @file_description,
               attributes: { maxlength: 10000 }, label: { text: "Description" },
-              hint: { text: "For example, 'Orange, 4 socket extension lead'" }
+              hint: { text: file_description_hint }
```
Comment (slorek) [inline psd-web/app/views/investigations/ts_investigations/_file_page.html.slim]: `file_description_hint || "For example, ‘Orange, 4 socket extension lead'’"`

File: psd-web/app/views/investigations/ts_investigations/_file_page.html.slim (hunk 1 above)
Comment (edwardhorsford) [reply psd-web/app/views/investigations/ts_investigations/_file_page.html.slim]: `.present` would only return true or false - whereas here I want it to return the value.

File: psd-web/app/views/investigations/ts_investigations/_file_page.html.slim (hunk 2 above)
Comment (edwardhorsford) [reply psd-web/app/views/investigations/ts_investigations/_file_page.html.slim]: I don't think slim supports this.

**Pull request merged**